src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ArrayBuffer.cs

```diff
@@ -128,5 +129,10 @@ public void EnsureAvailableSpace(int byteCount)
 
             Debug.Assert(byteCount <= AvailableLength);
         }
+
+        public void Grow()
```

Comment:
Maybe make this take in an int for the amount to Grow by?

Reply:
`EnsureAvailableSpace()` does that already -- `Grow()` is there when we don't care and just want the buffer doubled in size.